### PR TITLE
[INLONG-8008][Dashboard] Add default icon for different data nodes

### LINKED ISSUE #8008: [Improve][Dashboard] Add default icon for different data nodes
### Description

Add default icon for different data nodes, making it easier identifiable.

image

image

For example:
mysql ->
image

### InLong Component

InLong Dashboard

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by richardji202**
### Prepare a Pull Request

- Fixes #8008 

### Motivation

Add default icon for different data nodes

### Modifications

search and add default icon for different data nodes

The CheckCard component supports loading of local logo images
source
sink
hover tooltip

### Verifying this change

*(Please pick either of the following options)*

- [x] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### CODE REVIEW

**Comment by dockerzhang** [comment]
@richardji202 thanks for your contribution. Please add a rendering image for this PR.

File: inlong-dashboard/src/ui/components/CheckCard/index.tsx [hunk 1]
```diff
@@ -52,6 +54,25 @@ const CheckCard: React.FC<CheckCardProps> = ({ options, value, onChange, disable
     // eslint-disable-next-line
   }, [value]);
 
+  useEffect(() => {
+    // vite 不支持 require.context, 故使用import()动态获取每个类型的logo，模块是否存在由promise判断，模块缓存由esm处理
```
Comment (healchow) [inline inlong-dashboard/src/ui/components/CheckCard/index.tsx:58]: Please change the comment to English, thanks.

File: inlong-dashboard/src/ui/components/CheckCard/index.tsx [hunk 2]
```diff
@@ -72,8 +106,7 @@ const CheckCard: React.FC<CheckCardProps> = ({ options, value, onChange, disable
               bodyStyle={{ textAlign: 'center' }}
               className={disabled ? styles.cardDisabled : ''}
             >
-              <DatabaseOutlined style={{ fontSize: 20 }} />
-              <div>{options.find(item => item.value === currentValue)?.label}</div>
+              {renderContent(options.find(item => item.value === currentValue)?.label)}
```
Comment (leezng) [inline inlong-dashboard/src/ui/components/CheckCard/index.tsx:109]: Can you add a [tooltip](https://ant.design/components/tooltip)? Some logos have no clear meaning, for example when I hover to `File`.

File: inlong-dashboard/src/ui/components/CheckCard/index.tsx (hunk 1 above)
Comment (leezng) [inline inlong-dashboard/src/ui/components/CheckCard/index.tsx:58]: Please translate to english.

File: inlong-dashboard/src/ui/components/CheckCard/logo/Mongodb.png
Comment (healchow) [inline inlong-dashboard/src/ui/components/CheckCard/logo/Mongodb.png:1]: Can this file name change to `MongoDB.png`?

Comment (richardji202) [reply inlong-dashboard/src/ui/components/CheckCard/logo/Mongodb.png:1]: The dynamic loading of images is currently based on the "label" of this node, and the "label" of this node is "Mongodb". The configuration file ("@/src/plugins/sources/defaults/Mongodb.ts") is also "Mongodb". Please confirm how to adjust it.

Comment (richardji202) [reply inlong-dashboard/src/ui/components/CheckCard/logo/Mongodb.png:1]: I changed both the name of the PNG image and the label to "MongoDB".

Comment (healchow) [inline inlong-dashboard/src/ui/components/CheckCard/logo/Mongodb.png:1]: Sorry, but why do we need 2 images for MongoDB here?

Comment (richardji202) [reply inlong-dashboard/src/ui/components/CheckCard/logo/Mongodb.png:1]: Sorry, it seems that there was inconsistency between the remote repository and my local repository. It has been fixed now.

**Review by leezng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Pull request merged**